<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCVS_Interface</class>
 <widget class="QMainWindow" name="frmCVS_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1479</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpScanControls">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>1021</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Scan Controls</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="_lblEnvelopePrompt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Envelopes by Division</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="_lblCurvePrompt">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Location of Interest</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="_lblControlPrompt">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>20</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>System Control</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frmDivision">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QRadioButton" name="rbAD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>A Division</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbBD">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>B Division</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnREA">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Capture REA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLEA">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Capture LEA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSEA">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Capture SEA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHM">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Home Scanner</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSP">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Capture Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTL">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Toggle Laser</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTM">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Toggle Motor</string>
     </property>
    </widget>
    <widget class="QFrame" name="frmCurve">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QRadioButton" name="rbOutside">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Outside of Curve</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbInside">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Inside of Curve</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpScanData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Scan Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTableWidget" name="lstScanData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>361</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btnModifyScan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>371</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Modfiy Scan Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="_lblActiveScanPrompt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>450</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Active Scan:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblActiveScan">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>450</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpScanManagement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Scan Management</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="btnCreateScan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Create Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelectScan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>381</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Select Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveScan">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>400</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Save Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeleteScan">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>400</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Delete Scan</string>
     </property>
    </widget>
    <widget class="QTableView" name="lstProfiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>391</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpVisual">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>1031</width>
      <height>781</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Scan Vizualization</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1479</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_Scan_s">
     <property name="title">
      <string>Export Scan(s)</string>
     </property>
     <addaction name="actClearanceReport"/>
     <addaction name="actClearanceScan"/>
     <addaction name="actDataTable"/>
     <addaction name="actScanPoints"/>
    </widget>
    <addaction name="actNewScan"/>
    <addaction name="separator"/>
    <addaction name="actSaveScan"/>
    <addaction name="separator"/>
    <addaction name="menuExport_Scan_s"/>
    <addaction name="separator"/>
    <addaction name="actionUpload_to_Cloud"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionCalibrate"/>
    <addaction name="actReconnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSystem"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Open Existing</string>
   </property>
  </action>
  <action name="actCloud">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cloud Upload</string>
   </property>
  </action>
  <action name="actExportPlot">
   <property name="text">
    <string>Scan Visualization</string>
   </property>
  </action>
  <action name="actExportReport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clearance Scan Report</string>
   </property>
  </action>
  <action name="actExportData">
   <property name="text">
    <string>Scan Data Table</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>Clear Data</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="text">
    <string>Calibration</string>
   </property>
  </action>
  <action name="actConfigDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actConfigCustom">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actMQTTClient">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>MQTT Client</string>
   </property>
  </action>
  <action name="actDataView">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Data Editor</string>
   </property>
  </action>
  <action name="actSerialMonitor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>System Serial Monitor</string>
   </property>
  </action>
  <action name="actPOSXCAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scanner Horizontal Offset</string>
   </property>
  </action>
  <action name="actPOSYCAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scanner Vertical Offset</string>
   </property>
  </action>
  <action name="actMPUXCAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Gyroscope X Calibration</string>
   </property>
  </action>
  <action name="actMPUYCAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Gyroscope Y Calibration</string>
   </property>
  </action>
  <action name="actMPUZCAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Gyroscope Z Calibration</string>
   </property>
  </action>
  <action name="actMPUTCAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>MPU Temperature Calibration</string>
   </property>
  </action>
  <action name="actTFMDCAL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scanner Distance Calibration</string>
   </property>
  </action>
  <action name="actUnlockCalibration">
   <property name="text">
    <string>Unlock Calibration Wizard</string>
   </property>
  </action>
  <action name="actToggleMotor">
   <property name="text">
    <string>Toggle Motor Driver</string>
   </property>
  </action>
  <action name="actToggleLaser">
   <property name="text">
    <string>Toggle Laser</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actNewScan">
   <property name="text">
    <string>New Scan</string>
   </property>
  </action>
  <action name="actionImport_Scans">
   <property name="text">
    <string>Import Scans</string>
   </property>
  </action>
  <action name="actSaveScan">
   <property name="text">
    <string>Save Scan</string>
   </property>
  </action>
  <action name="actionSave_Scan_as">
   <property name="text">
    <string>Save Scan as</string>
   </property>
  </action>
  <action name="actionUpload_to_Cloud">
   <property name="text">
    <string>Upload to Cloud</string>
   </property>
  </action>
  <action name="actClearanceReport">
   <property name="text">
    <string>Clearance Report (PDF)</string>
   </property>
  </action>
  <action name="actClearanceScan">
   <property name="text">
    <string>Clearance Scan (JPG)</string>
   </property>
  </action>
  <action name="actDataTable">
   <property name="text">
    <string>Data Table (CSV)</string>
   </property>
  </action>
  <action name="actScanPoints">
   <property name="text">
    <string>Scan Points (CSV)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
